light-4j/0017-jdk11-branch-release.md

```diff
@@ -0,0 +1,34 @@
+### Summary
+
+The light platform 1.5.x is built on top of JDK 8 and it is production ready and we will support it until Java 8 is not supported. After JDK 11 is release, we have another jdk11 branch that contains the code on top of JDK 11. This branch will be the 2.0.X release and any new feature and defect fixes will be merged to two branches develop and jdk11 at the same time. After the release for JDK8, the code will be merged from develop to master. For JDK11, the code will be merged from jdk11 to master-jdk11. Once majority of our customers moves to the JDK11, we are going to switch the develop and master branch to JDK 11 and JDK8 will become jdk8 and master-jdk8 for maintance. 
+
+Now, for a period of time, we need to have two releases from to diferent branches. That means we have to change our release process to automate it. 
```

Comment:
> from two different branches...

Reply:
I would say that there will be 2 codebases, and they will evolve separately.
Therefore, individual branches for Java 8 and 11 would be required.

Although functionality could be similar on both versions, say a new module performing a specific task, one might leverage Java 11 and therefore would be best served to be committed to its individual branch

Reply:
There have to be two branches. In the beginning, we want to encourage the same PR codebase for both jdk8 and jdk11; however, we have to leverage some of the jdk11 features eventually. In that case, there might be two separate PRs to fix the same thing for both jdk8 and jdk11.